test/uploader_spec.js

```diff
@@ -713,24 +713,33 @@ describe("uploader", function () {
       });
     });
   });
-  it("should support unsigned uploading using presets", function () {
+  it("should support unsigned uploading using presets", async function () {
     this.timeout(helper.TIMEOUT_LONG);
-    let presetName;
-    return cloudinary.v2.api.create_upload_preset({
+
+    let preset = await cloudinary.v2.api.create_upload_preset({
       folder: "upload_folder",
       unsigned: true,
       tags: UPLOAD_TAGS,
-    }).then(function (preset) {
-      presetName = preset.name;
-      return cloudinary.v2.uploader.unsigned_upload(IMAGE_FILE, preset.name, {
-        tags: UPLOAD_TAGS,
-      });
-    }).then(function ({ public_id }) {
-      expect(public_id).to.match(/^upload_folder\/[a-z0-9]+$/);
-    }).finally(function () {
-      return cloudinary.v2.api.delete_upload_preset(presetName);
+    }).catch((err) => {
+      console.log(err);
+      throw new Error('Create upload preset call failed');
```

Comment:
Let's set a convention of those error messages - "[function name] failed"
in this case - "create_upload_preset failed"